src/networks/mod.rs

```diff
@@ -182,14 +182,6 @@ struct DrandPoint<'a> {
 pub struct ChainConfig {
     pub network: NetworkChain,
     pub genesis_cid: Option<String>,
-    #[cfg_attr(test, arbitrary(gen(
-        |g: &mut quickcheck::Gen| {
-            let addr = std::net::Ipv4Addr::arbitrary(&mut *g);
-            let n = u8::arbitrary(g) as usize;
-            vec![addr.into(); n]
-        }
-    )))]
-    pub bootstrap_peers: Vec<Multiaddr>,
```

Comment:
I think we should keep `bootstrap_peers` as a field, and exclude `ChainConfig` in its entirety from the configuration file.

Reply:
Done.